docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -447,154 +390,132 @@ where t is the period for which transition the is estimated (a year) eg. 365 day
+So...
 
-  k = :math:`\frac{\text{number of incident cases}}{\text{number of prevalent cases}}` see [Isanaka_2021]_ for full proof and equations. 
+  :math:`365/k = duration_\text{treated SAM} * C * E_\text{SAM} + duration_\text{untreated SAM} * (1 - C * E_\text{SAM})`
 
-  Number of incident cases = :math:`\frac{\text{annual programme admissions}}{\text{treatment coverage}}`
+  ...
 
-  1) From Isanaka 2021, the incidence correction factor for Ethiopia is k = 6.7(95%CI:5.3-8.4) for a baseline population SAM treatment coverage of 48.8(95%CI 37.4 to 60.4)
-  2) k = 6.7(95%CI:5.3-8.4) =  :math:`\frac{365}{SAM\_duration (in days)}` 
-  3) SAM_duration =  :math:`\frac{365}{6.7}` = 54.5 days
-  4) :math:`54.5 = \frac{365}{r2_{ux} \times (1-CE) + t1_{sam} \times (CE) + death\_rate_{age|sex}}` 
-  5) Solve for :math:`r2_{ux}` for for each age/sex strata with age/sex specific death rate
- 
- where C = 48.8% (95%CI:37.4-60.4), E = 70% (95%CI:64-76). Note that k = 6.7 corresponds to the current treatment coverage estimate in Isanaka 2021 (see appendix) which is C = 48.8% (what we call baseline). We are further making an assumption that the baseline efficacy of the program is E = 70%. Hence from these baseline assumptions, we are able to derive a r2_ux. Also note that only the treatment coverage C is necessary to calculate the number of incidence cases.  
+  :math:`duration_\text{untreated SAM} = \frac{365/k - duration_\text{treated SAM} * C * E_\text{SAM}}{1 - C * E_\text{SAM}}`
 
-  For example, if 
+.. note::
 
-  - :math:`death\_rate_{age|sex}` = 0.015 per person year for age group 4 female 
-  - :math:`r2_{ux} \times (1 - 0.488  \times 0.7) = \frac{365}{54.5} - (\frac{365}{48.3}  \times 0.488  \times 0.7) - 0.015`
-  - :math:`r2_{ux} \times (1 - 0.488  \times 0.7) = 6.697 - 2.581 - 0.015`
-  - :math:`r2_{ux} \times (1 - 0.488  \times 0.7) = 4.101`
-  - :math:`r2_{ux} =  4.101 / 0.6584 = 6.2287`
-  - Duration of untreated SAM = 365 / 6.2287 = 58.6 days for age group 4 female
+  A note on the incidence correction factor, :math:`k`: 
 
 
-.. important::
+    The incidence correction factor :math:`k` is = :math:`\frac{t}{\text{average duration of disease}}`, where :math:`t` is the period for which incidence is estimated (a year) eg. 365 days using days as the unit 
 
-  In this first run that Rajan is building (August-18-21), we are not modelling treatment in the first 28 days of life. If we do model treatment in this first 28 days, the prevalence of wasting by the time they reach post-neonatal age groups in the scale-up scenario will be different (reduced). We should think through the relationship between wasting and LBWSG in the neonatal age groups and see if we want to add treatment back into the first month of life. 
+    k = :math:`\frac{\text{number of incident cases}}{\text{number of prevalent cases}}` see [Isanaka_2021]_ for full proof and equations. 
```

Comment:
Let's put some details on how k = 6.7 is estimated in this doc.  it is sneaky and interesting